mobile/cmp/panel/Panel.js

```diff
@@ -10,6 +10,7 @@ import {castArray, omitBy} from 'lodash';
 import {elemFactory, HoistComponent, LayoutSupport} from '@xh/hoist/core';
 import {vbox, vframe} from '@xh/hoist/cmp/layout';
 import {toolbar} from '@xh/hoist/mobile/cmp/toolbar';
+import {loadingIndicator} from '@xh/hoist/desktop/cmp/loadingindicator';
```

Comment:
We should never have imports from the desktop package into mobile!  

Since the LoadingIndicator is relying on the spinner out of Blueprint, it's also problematic to promote to a cross-platform component as it is.  For this iteration, I would be OK leaving it out of mobile unless we knew we had an immediate use for it.  If/when we do need it, we could look to use a spinner that's not sourced from Blueprint.

Reply:
We don't have a mobile spinner-like thing?  Don't we need one?

Reply:
(but am fine with leaving it off mobile to start)

Reply:
We have a mobile spinner called `progressCircular`, that's being used as part of the mobile mask cmp - could use that if we continue with the mobile route

Reply:
I've made a mobile loadingIndicator component. and switched the mobile panel to use it.   I have not switched the mobile panel to use the mobile Mask.  Tom said he would do that in a separate PR.  I don't want to get in his way on that.